src/main/java/com/kryptnostic/conductor/rpc/ConductorSparkApi.java

```diff
@@ -31,4 +40,19 @@ QueryResult getAllEntitiesOfEntitySet(
      * @return QueryResult of UUID's matching the lookup request
      */
     QueryResult getFilterEntities( LookupEntitiesRequest request );
+
+	Boolean submitEntitySetToElasticsearch( EntitySet entitySet, List<PropertyType> propertyTypes, Principal principal );
```

Comment:
These should be in their own interface. For now it's fine if `ConductorSparkImpl` implements both interfaces.